arrows/serialize/json/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+# Build / Install plugin for serialization
+
+set( headers_public
+
+  bounding_box.h
+  detected_object.h
+  detected_object_set.h
+  detected_object_type.h
+  )
+
+set( sources
+
+  bounding_box.cxx
+  detected_object.cxx
+  detected_object_set.cxx
+  detected_object_type.cxx
+  )
+
+kwiver_add_library( kwiver_serialize_json
+  ${headers_public}
+  ${sources}
+  )
+
+target_link_libraries( kwiver_serialize_json
+  PUBLIC               vital
+                       vital_vpm
+  PRIVATE              vital_algo
```

Comment:
I think vital_algo is in an included header, which might require it to be PUBLIC. Hard to follow the paper trail though, could be wrong.